# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eg/slang_path.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./egcmplte.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egcmplte.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <string.h>
 #include <limits.h>
 #include <dirent.h>
--- ./eg.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./eg.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <stdlib.h>
 #include <unistd.h>
 #include <sys/utsname.h>
--- ./egdir.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egdir.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
--- ./egdraw.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egdraw.c	2004-03-04 19:56:33.000000000 -0300
@@ -19,7 +19,7 @@
 
 */
 
-#include <slang/slang.h>
+#include <slang.h>
 #include <stdio.h>
 #include <stdlib.h>
 
--- ./eggetfld.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./eggetfld.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <ctype.h>
 #include <string.h>
 #include <stdlib.h>
--- ./eghelp.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./eghelp.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 
 #include "eg.h"
 
--- ./eglib.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./eglib.c	2004-03-04 19:56:33.000000000 -0300
@@ -21,7 +21,7 @@
 
 #include <pwd.h>
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <errno.h>
 #include <ctype.h>
 #include <string.h>
--- ./egmenu.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egmenu.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <stdlib.h>
 #include <string.h>
 
--- ./egnavgte.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egnavgte.c	2004-03-04 19:56:33.000000000 -0300
@@ -19,7 +19,7 @@
 
 */
 
-#include <slang/slang.h>
+#include <slang.h>
 #include <stdio.h>
 #include <ctype.h>
 #include <limits.h>
--- ./egregex.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egregex.c	2004-03-04 19:56:33.000000000 -0300
@@ -25,7 +25,7 @@
 #include <sys/types.h>
 #endif
 #include <regex.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <stdlib.h>
 
 #include "eg.h"
--- ./egscreen.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egscreen.c	2004-03-04 19:56:33.000000000 -0300
@@ -20,7 +20,7 @@
 */
 
 #include <stdio.h>
-#include <slang/slang.h>
+#include <slang.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
--- ./egsigs.c.orig	2004-03-04 19:56:33.000000000 -0300
+++ ./egsigs.c	2004-03-04 19:56:33.000000000 -0300
@@ -22,7 +22,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <signal.h>
-#include <slang/slang.h>
+#include <slang.h>
 
 #include "eg.h"
 
